examples/inference/stable_diffusion/sd3_generation.py

```diff
@@ -51,12 +54,17 @@ def infer(args):
     # ==============================
     coordinator.print_on_master(f"Generating...")
     out = engine.generate(prompts=[args.prompt], generation_config=DiffusionGenerationConfig())[0]
-    out.save("cat.jpg")
+    if dist.get_rank() == 0:
+        out.save(f"cat_parallel_size{dist.get_world_size()}.jpg")
     coordinator.print_on_master(out)
 
 
 # colossalai run --nproc_per_node 1 examples/inference/stable_diffusion/sd3_generation.py -m MODEL_PATH
-# colossalai run --nproc_per_node 1 examples/inference/stable_diffusion/sd3_generation.py -m "stabilityai/stable-diffusion-3-medium-diffusers" --tp_size 1
+# CUDA_VISIBLE_DEVICES_set_n_least_memory_usage 1 && colossalai run --nproc_per_node 1 examples/inference/stable_diffusion/sd3_generation.py -m "stabilityai/stable-diffusion-3-medium-diffusers" --tp_size 1
```

Comment:
`CUDA_VISIBLE_DEVICES_set_n_least_memory_usage` is only set and exposed in `examples/inference/llama/run_benchmark.sh` and should not be brought as an instruction command